rotkehlchen/data/all_assets.json

```diff
@@ -10526,6 +10526,14 @@
         "symbol": "YACC",
         "type": "own chain"
     },
+    "YAM": {
+        "ethereum_address": "0x0e2298E3B3390e3b945a5456fBf59eCc3f55DA16",
+        "ethereum_token_decimals": 18,
+        "name": "YAM",
+        "started": 1597172400,
```

Comment:
This should be 1597121253. According to the deployment transaction timestamp https://etherscan.io/tx/0x1b96008716c88a7bda17a1ba479af2725dc5ae81a590835402d081a74132e35c

Reply:
I took the launch date from their medium post: https://medium.com/@yamfinance/yam-finance-d0ad577250c7
I believe it is more accurate because people only were able to interact with the contracts at that point in time.

Reply:
I see. Doesn't matter though as the contract deployment is what we use for all ethereum tokens.